modules/ROOT/pages/project-best-practices-index.adoc

```diff
@@ -1,33 +1,51 @@
 = Project best practices
-:description: Helpful experience we share for the good of your project.
-
-Helpful experience we share for the good of your project.
-
-blabla
-[.card-section]
-== Worth trying
-
-[.card.card-index]
---
-xref:teamwork.adoc[[.card-title]#Create a multi-skilled team# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:lifecycle-and-profiles.adoc[[.card-title]#Project phases# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:agility.adoc[[.card-title]#Agility# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:software-craftsmanship.adoc[[.card-title]#Software craftsmanship# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:devops.adoc[[.card-title]#DevOps# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
+:description: Helpful experience we share for the good of Bonita projects.
+
+Helpful experience we share for the good of Bonita projects.
+
+This section contains two chapters today, but we will keep improving to share more.
```

Comment:
This sentence brings nothing to the user, I suggest to remove it.
```suggestion
```

Reply:
Ah..... ok. I agree to take it away as I am not sure WHEN we will add more content. It will not look good in 1 year or so.